integration-cli/docker_cli_commit_test.go

```diff
@@ -37,6 +38,21 @@ func (s *DockerSuite) TestCommitWithoutPause(c *check.C) {
 	dockerCmd(c, "inspect", cleanedImageID)
 }
 
+func (s *DockerSuite) TestCommitWithoutCgroupFreeze(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	outId, _ := dockerCmd(c, "run", "-i", "-d", "busybox")
+
+	containerID := strings.TrimSpace(outId)
+	output, _ := dockerCmd(c, "exec", containerID, "cat /proc/self/cgroup | grep freeze | awk -F : '{print $3}'")
+
+	freezePath := strings.TrimSpace(output)
+	os.RemoveAll(strings.Join([]string{"/sys/fs/cgroup/freeze", freezePath}, ""))
```

Comment:
This just seems dangerous in the greater scope of things (all the other tests to run).